com.unity.render-pipelines.universal/Editor/Converter/Converters.cs

```diff
@@ -0,0 +1,226 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace UnityEditor.Rendering.Universal
+{
+    /// <summary>
+    /// Used for Converting in Batch Mode.
```

Comment:
This doesn't actually describe the enum. For referring to the batch mode, use `<seealso>` tag to refer to the actual function.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#seealso